data/skeletons/lalr1.cc

```diff
@@ -1355,7 +1385,9 @@ b4_dollar_popdef])[]dnl
     if (!yyla.empty ())
       {
         symbol_number_type yytoken = yyla.type_get ();
-        yyarg[yycount++] = yytname_[yytoken];]b4_lac_if([[
+        yyarg[yycount++] = ]b4_parse_error_case(
+              [verbose], [[yytname_[yytoken]]],
+              [[yysymbol_name(yytoken)]])[;]b4_lac_if([[
```

Comment:
Don't forget the space before the paren please!  Several occurrences with `yysymbol_name`.

Reply:
I went over all the commits again and I hope I caught all places.

Note that `_` and `YY_` are consistently called without space before the paren. I did not change the space for those two functions/macros

Reply:
Yup, that's the recommended style for gettext.